### ISSUE TITLE
Relations point to itself

### ISSUE THREAD

**Issue by rafaelcorreiapoli** [graphcool/prisma#1841]
# Bug Report

**Current behavior**
I have this schema:
```
type Technology {
  id: ID! @unique
  name: String!
  childTechnologies: [Technology!]! @relation(name: "ChildTechnologies")
  parentTechnologies: [Technology!]! @relation(name: "ChildTechnologies")
}
```

When trying to add a child techB to the parent techA:
```
ctx.db.mutation.updateTechnology({
      where: {
        id: "techA"
      },
      data: {
        childTechnologies: {
          connect: {
            id: "techB"
          }
        }
      }
    }, info)
```
The final result is:
```
{
  "data": {
    "technologies": [
      {
        "id": "techA",
        "name": "My Technology Updated",
        "childTechnologies": [
          {
            "id": "techB"
          }
        ],
        "parentTechnologies": [
          {
            "id": "techB"
          }
        ]
      },
      {
        "id": "techB",
        "name": "Another Technology",
        "childTechnologies": [],
        "parentTechnologies": []
      }
    ]
  }
}
```

**Reproduction**

https://github.com/rafaelcorreiapoli/prisma-playground
Just run `yarn test`

**Expected behavior?**

The result should be:
```
{
  "data": {
    "technologies": [
      {
        "id": "techA",
        "name": "My Technology Updated",
        "childTechnologies": [
          {
            "id": "techB"
          }
        ],
        "parentTechnologies": []
      },
      {
        "id": "techB",
        "name": "Another Technology",
        "childTechnologies": [],
        "parentTechnologies": [
             {
                "id": "techA"
            }]
      }
    ]
  }
}
```

**Comment by telaoumatenyanis** [comment]
Hello,
this is a blocking issue for us too, it was not happening with GraphCool, but it does with Prisma, thus delaying our migration.

**Comment by crubier** [comment]
Hi, this is a big problem for me at the moment, it really prevents us to use Prisma in production.

**Comment by rafaelcorreiapoli** [comment]
Yes, I was willing to use Prisma on a big project and this prevented me from doing it
I think that this issue should be treated with priority :)
Anyway, thanks for the awesome job guys!

**Comment by do4gr** [comment]
Hey, @rafaelcorreiapoli, @telaoumatenyanis , @crubier  thanks for reporting this and providing a simple example. I can reproduce this and will look into this.

**Comment by marktani** [comment]
This is now fixed and available in `1.4.0` 🙂Thanks a lot for your help!
